user/tests/wiring/serial_loopback/serial.cpp

```diff
@@ -24,23 +24,14 @@
  */
 
 #include "application.h"
-#if (PLATFORM_ID == 0)
-#include "Serial2/Serial2.h"
-#endif
 #include "unit-test/unit-test.h"
 
 /*
  * Serial1 Test requires TX to be jumpered to RX as follows:
- * valid for both Core and Photon
+ * on the photon Photon
```

Comment:
photon Photon

Reply:
The "photon Photon" is new... I guess you haven't heard of it. :smile: :ear: